src/VCR/Videorecorder.php

```diff
@@ -95,13 +95,13 @@ public function getEventDispatcher()
     /**
      * Dispatches an event to all registered listeners.
      *
-     * @param string $eventName The name of the event to dispatch.
      * @param Event $event The event to pass to the event handlers/listeners.
+     * @param string $eventName The name of the event to dispatch.
      * @return Event
      */
-    private function dispatch($eventName, Event $event = null)
+    private function dispatch(Event $event, $eventName = null)
```

Comment:
Should I add a return type hint to match the style in the 1.5 branch?

Reply:
If possible, yes please.